rllib/tuned_examples/cql/pendulum-cql-fake-gpus.yaml

```diff
@@ -0,0 +1,48 @@
+# Given a SAC-generated offline file generated via:
+# rllib train -f tuned_examples/sac/pendulum-sac.yaml --no-ray-ui
+
+# Pendulum CQL can attain ~ -300 reward in 10k from that file.
```

Comment:
nit, this comment reads a bit weird, our stop criteria is to read -700 in 800K steps?